doc/source/conf.py

```diff
@@ -104,5 +104,5 @@ def build_doxygen(app):
 # -- Setup hooks -------------------------------------------------------------
 
 def setup(app):
-    app.add_stylesheet('theme_overrides.css')
+    app.add_css_file('theme_overrides.css')
```

Comment:
`add_stylesheet` was deprecated in favor of `add_css_file`.  See, for example: https://github.com/sphinx-doc/sphinx/issues/7747